lib/is_external_link.js

```diff
@@ -18,17 +10,19 @@ const urlObj = (str) => {
 
 function isExternalLink(url) {
```

Comment:
suggest to rename to `path` or `input` to not confuse with `new URL()`.

Reply:
`input` might be a good idea.